lib/src/Uncertainty/Algorithm/Analytical/SORMResult.cxx

```diff
@@ -409,10 +409,10 @@ void SORMResult::load(Advocate & adv)
   adv.loadAttribute( "sortedCurvatures_", sortedCurvatures_ );
   adv.loadAttribute( "isAlreadyComputedSortedCurvatures_", isAlreadyComputedSortedCurvatures_ );
   adv.loadAttribute( "eventProbabilityBreitung_", eventProbabilityBreitung_ );
-  adv.loadAttribute( "eventProbabilityHohenBichler_", eventProbabilityHohenBichler_ );
+  adv.loadAttribute( "eventProbabilityHohenbichler_", eventProbabilityHohenbichler_ );
```

Comment:
you must provide a backward-compatible mecanism to load old xml files

Reply:
Where does @vchabri need to put this code, @jschueller? Directly after this line?

Reply:
as replacement, (with newname=eventProbabilityHohenbichler_, oldname=eventProbabilityHohenBichler_)

Reply:
Done.